docs/Makefile

```diff
@@ -1,16 +1,18 @@
 PREFIX ?= /usr/local
 DATADIR ?= ${PREFIX}/share
 MANDIR ?= $(DATADIR)/man
-MANDOWN ?= mandown
+GO ?= go
+GOBIN ?= $(GOPATH)/bin
+GOMD2MAN ?= $(shell command -v go-md2man || echo '$(GOBIN)/go-md2man')
 
 docs: $(patsubst %.md,%,$(wildcard *.1.md))
 
 %.1: %.1.md
-	$(MANDOWN) $< > $@
+	$(GOMD2MAN) -in $^ -out $@
 
-.PHONY: .install.mandown
-.install.mandown:
-	cargo install mandown
+.PHONY: .install.md2man
+.install.md2man:
+	$(GO) install github.com/cpuguy83/go-md2man/v2@latest
```

Comment:
Wasn't sure whether to simply remove this and assume `go-md2man` was installed, or update this convenience target to install it. Happy to simply drop it if preferred.

Reply:
Doesn't this assume the caller has `$GOPATH/bin` in their `$PATH`?  Seeing as this is not a golang project, it might be nice to at least mention that in a comment.

Reply:
To an extent. If you look at [line 6](https://github.com/containers/netavark/pull/771/files#diff-cdec797dddf8fc387304501d6bd4f318e1e9067c6fae012dfa8b69fd85fd3248R6), you can see we use `command` to check for it in `$PATH`, falling back to `$(GOBIN)/go-md2man`. The line above sets a default for `$GOBIN` in case it's unset on the environment.

Reply:
I don't see any default for `$GOPATH`, so `$GOBIN` will default to `/bin` which is probably wrong.

Reply:
I think we can just assume it to be in $PATH, on go dev envronment it is very likely that gobin is in $PATH anyway and for CI systems it should really be installed $PATH.

as for line 6 we can simply leave it as `go-md2man` like we did with mandown.

Reply:
@Luap99 Should I go ahead and remove `.install.md2man` then?